src/test/java/org/broadinstitute/hellbender/tools/walkers/filters/VariantFiltrationIntegrationTest.java

```diff
@@ -272,4 +283,23 @@ public void testFilteringZfromFORMATAndFailMissing() throws IOException {
 
         spec.executeTest("testFilteringZfromFORMATAndFailMissing", this);
     }
+
+    @Test
+    public void testFilteredAndPassBecomeUnfiltered() throws IOException {
```

Comment:
I think this would be a better test if it ran twice both INVALIDATE_PREVIOUS_FILTERS_LONG_NAME = true and false.